src/devtools-connector/hot-code-reloader.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * @license
+ * Copyright (c) 2018 Google Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import {Arc} from '../runtime/arc.js';
+import {ArcDevtoolsChannel, DevtoolsMessage} from './abstract-devtools-channel.js';
+
+export class HotCodeReloader {
```

Comment:
Add class header documentation -- what is the purpose of this class? Ex. "Monitors particle reload events for all particles in an arc and reloads them if their source file changes."

Reply:
Added!